libbeat/outputs/fileout/file.go

```diff
@@ -74,10 +71,14 @@ func makeFileout(
 
 func (out *fileOutput) init(beat beat.Info, c fileOutConfig) error {
 	var path string
+	configPath, runErr := c.Path.Run(time.Now())
```

Comment:
I didn't look into this, but does the time zone of this matter? Should we call `time.Now().UTC()` so that the behavior is always consistent?

Reply:
good idea